app/services/action_builder.rb

```diff
@@ -0,0 +1,23 @@
+module ActionBuilder
```

Comment:
I think that a mixin like this is a great time to use shared examples for the spec. That's what I did for the fundraiser and petition plugins when I created the HasForm concern.